webextensions/manifest/theme.json

```diff
@@ -464,7 +464,9 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": "62"
+                  "version_added": "62",
+                  "version_removed": "89",
+                  "notes": "This theme property has been deprecated in Firefox 89. See the <a href='https://blog.mozilla.org/addons/2021/04/19/changes-to-themeable-areas-of-firefox-in-version-89/'>related blogpost</a> for more details."
```

Comment:
Nit: maybe include the blog title in the link text to make it easier to see what the blog is about without hovering over the link?